config/values-schema.yml

```diff
@@ -0,0 +1,18 @@
+#@data/values-schema
+---
+#@schema/desc "The namespace in which to deploy kapp-controller"
+namespace: kapp-controller
+#@schema/desc "Whether to create namespace specified for kapp-controller"
+create_namespace: true
+#@schema/desc "packaging global namespace value"
```

Comment:
We should be consistent about starting the description with a Capital letter. And some of these don't really describe the values so much as a long version of the key, e.g.:

```
#@schema/desc "dangerous enable pprof flag"
dangerous_enable_pprof: false
```

I still have to go read the documentation to find out why or why not to enable this flag. It would be helpful if the values were documenting as best we could such that you don't need to go too far to know what each key is for

Reply:
Sure. Got the description from `dev.md` it is used to enable profiling by setting it to true.